googletest/docs/FAQ.md

```diff
@@ -211,14 +211,15 @@ combined. Therefore we want to invest more in the matchers than in the
 
 ## Does Google Test support running tests in parallel? ##
 
-Test runners tend to be tightly coupled with the build/test
-environment, and Google Test doesn't try to solve the problem of
-running tests in parallel.  Instead, we tried to make Google Test work
-nicely with test runners.  For example, Google Test's XML report
-contains the time spent on each test, and its `gtest_list_tests` and
-`gtest_filter` flags can be used for splitting the execution of test
-methods into multiple processes.  These functionalities can help the
-test runner run the tests in parallel.
+Test runners tend to be tightly coupled with the build/test environment, and
+Google Test doesn't try to solve the problem of running tests in parallel. An
+external tool has however been formulated to help with running tests in parallel,
```

Comment:
Please make it clear that the external tool is another open source project.
See also https://github.com/google/googletest#related-open-source-projects